tests/src/FunctionalJavascript/ModuleIntegrationTest.php

```diff
@@ -542,4 +542,67 @@ protected function liveblogSetBody(DocumentElement $page, $body) {
     );
   }
 
+  /**
+   * Testing integration of "AMP" module and theme.
+   */
+  public function testAmpIntegration() {
```

Comment:
My plan is to have module with test content and switch tests to use it. It will probably be based on demo content in first hand, but later it will be easier to add when something is needed.